packages/story-editor/src/components/library/panes/shopping/shoppingPane.js

```diff
@@ -269,20 +290,32 @@ function ShoppingPane(props) {
             <ProductSort onChange={onSortBy} value={`${orderby}-${order}`} />
           </Row>
         )}
-        {isLoading && (
-          <Loading>
-            <Spinner>
-              <CircularProgress size={24} />
-            </Spinner>
-          </Loading>
+        {!loaded && isLoading && (
+          <LoadingContainer>
+            <LoadingSpinner {...loadingSpinnerProps} />
+          </LoadingContainer>
         )}
-        {loaded && !isLoading && products?.length > 0 && (
-          <ProductList
-            isMenuFocused={isMenuFocused}
-            onClick={onClick}
-            products={products}
-            onPageProducts={currentPageProducts}
-          />
+        {loaded && products?.length > 0 && (
+          <>
+            <ProductList
+              isMenuFocused={isMenuFocused}
+              onClick={onClick}
+              products={products}
+              onPageProducts={currentPageProducts}
+            />
+            <InfiniteScroller
+              allDataLoadedMessage={allDataLoadedMessage}
+              allDataLoadedAriaMessage={__(
+                'Products are loaded',
```

Comment:
Testing with VoiceOver... on the initial load it seems "Products are loaded" get announced.

1) When no results turn up ... that doesn't seem to get announced.

![Screen Shot 2022-06-14 at 5 56 45 AM](https://user-images.githubusercontent.com/62242/173550725-7d73e41f-6a59-4234-be2a-527dac1a2bac.png)

2) I also get Loading more products announced when the search is already done.

![Screen Shot 2022-06-14 at 5 56 09 AM](https://user-images.githubusercontent.com/62242/173550865-73846880-26cb-4583-91b6-391e8712f91b.png)

Reply:
Can you create that issue?

Looks like `InfiniteScroller` should probably also use `useLiveRegion` for the announcements instead of the custom `AriaOnlyAlert`.